webpack/react_app/components/job_invocations/index.js

```diff
@@ -0,0 +1,47 @@
+import { groupBy, isUndefined } from 'lodash';
+import { connect } from 'react-redux';
+import React from 'react';
+import PieChart from 'foremanReact/components/common/charts/PieChart';
+import AggregateStatus from './AggregateStatus/index.js';
+
+import * as JobInvocationActions from '../../redux/actions/job_invocations';
+
+class JobInvocationContainer extends React.Component {
+  componentDidMount() {
+    const { startJobInvocationsPolling, data: { url } } = this.props;
+
+    startJobInvocationsPolling(url);
+  }
+
+  render() {
+    const {
+      job_invocations,
+      statuses,
+    } = this.props;
+
+    return (
+      <div id="job_invocations_chart_container">
+        <PieChart
```

Comment:
you should consider using patterfly-react chart components directly? (see https://github.com/theforeman/foreman/pull/5137 for background)

Reply:
Makes sense, but after that PR is merged we can update REX here. My main concern is that this PR is already only 1.17 compatible because of Axios, and using Patternfly donut chart from that PR would make it 1.18